src/stories/plans/userInfo/index.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta, Story } from '@storybook/addon-docs';
+
+<Meta title="Plans/UserInfo/Docs" />
+
+# UserInfo
+
+** Shows information about a User installing a Plan.**
+
+## Overview
+
+The **UserInfo** component is an
+[SLDS Card](https://react.lightningdesignsystem.com/components/cards/) used to
+display information about the Salesforce Org for installing a **Plan**.
+
+## States
+
+On a Product page for a Plan that requires connecting to an existing Org, before
+a User has connected the **UserInfo** component shows an illustration and text
+that reads "Not Connected to Salesforce."
+
+<Story id="plans-userinfo-examples--disconnected-user-info-component" />
+
+Once a User connects to a Salesforce Org, the component shows the Org owner's
+email, Org name, and Org type, if the information is provided. The header reads
+"Connected to Salesforce" and a message explains for how long the User's
+credentials will be valid. A message in the footer provides a link to a dropdown
+menu that gives the User the ability to log in with a different Org.
+
+<Story id="plans-userinfo-examples--connected-user-info-component" />
+
+If a Plan allows connecting to Salesforce through a temporary Scratch Org, the
+**UserInfo** component shows the number of days before the Org will expire, and
+a message warning what happens when the Org expires.
+
+<Story
+  id="plans-userinfo-examples--org-user-info-component&args=scratchOrg:Scratch+Org+not+yet+created"
+  height="250px"
+/>
+
+Once a Scratch Org has been created, the component displays the exact date when
+the Org will expire.
+
+<Story
+  id="plans-userinfo-examples--org-user-info-component&args=scratchOrg:Scratch+Org+created"
+  height="250px"
+/>
```

Comment:
Its not important, but is there any reason these are more narrow than the two cards cards above these?

Reply:
I think because the first two are rendered inline, while the latter two are rendered in iframes (and have to be). We could try rendering all of them in iframes...

Reply:
@jgerigmeyer @stacyk set the first two stories in the docs to [`inline={false}`](https://storybook.js.org/docs/react/writing-docs/doc-blocks#inline-rendering) and that adds the iframe and gets all the stories to be the same width

![bild](https://user-images.githubusercontent.com/41588129/141124559-41ec68e5-4a93-4120-9906-5bbeba9d2825.png)